compiler/optimizer/LoopReducer.cpp

```diff
@@ -1088,23 +1088,24 @@ TR_LoopReducer::generateArraycopy(TR_InductionVariable * indVar, TR::Block * loo
    if (arraycopyLoop.getStoreAddress()->getIncrement() < 0)
       //&& (storeAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol() == loadAddr->getFirstChild()->getSymbol()->getRegisterMappedSymbol()))
       {
-      if (comp()->cg()->getSupportsPrimitiveArrayCopy())
-         arraycopy->setBackwardArrayCopy(true); /* bit available only to primitive arraycopy */
```

Comment:
> In fact I've checked all the codegens we currently support. No codegen ever checks for the isBackwardArrayCopy flag. They all check for isForwardArrayCopy and if this returns false we assume its a backwards array copy

I'm still trying to understand the surface area of all of these flags, but one thing to note is that `isBackwardArrayCopy` and `isForwardArrayCopy` are not independent. Setting one unsets the other. Furthermore, it's not obvious that the interpretation of `isForwardArrayCopy` is the same on all platforms. A comment at the start of the z `arraycopyEvaluator` describes its interpretation in terms that make claims about "390 codegen" rather than about the meaning of the flag more broadly.